sdk/src/connections/target/adsd3500_sensor.cpp

```diff
@@ -1099,9 +1099,11 @@ aditof::Status Adsd3500Sensor::adsd3500_read_payload_cmd(uint32_t cmd,
         return Status::GENERIC_ERROR;
     }
 
-    if (cmd == 0x13 || cmd == 0x19) {
+    if (cmd == 0x13)
```

Comment:
using 'if' then 'else if' is a little more efficient than 2 'ifs'.

Reply:
Fixed.